sumo/cli/dosplot.py

```diff
@@ -192,9 +187,28 @@ def dosplot(filename=None, code='vasp', prefix=None, directory=None,
             else:
                 logging.error('ERROR: Too many *.bands files found!')
                 sys.exit()
-        dos = sumo.io.castep.read_tdos(bands_file, gaussian=gaussian,
-                                       emin=xmin, emax=xmax)
-        pdos = {}
+        pdos_file = bands_file.replace('.bands', '.pdos_bin')
+        cell_file = bands_file.replace('.bands', '.cell')
+        pdos_file = pdos_file if os.path.isfile(pdos_file) else None
+        cell_file = cell_file if os.path.isfile(cell_file) else None
+
+        # Check if both pdos_bin and cell files are present.
+        # If not, we cannot plot the PDOS
+        if pdos_file is not None:
+            if cell_file is None:
+                logging.info("Plotting PDOS requires the cell file to be present, falling back to TDOS.")
+                pdos_file = None
+            else:
+                logging.info(f"Found PDOS binary file {pdos_file}, including PDOS in the plot.")
```

Comment:
f-strings are usable as of Python 3.6. Officially v1.3.0 is the last Sumo version to support python 3.5, but setup.py still supports it. So if this is the first PR to include f-strings, I think it is also time to update setup.py and Travis to actively drop 3.5! @utf is that ok?

Reply:
(This is not the only f-string in the change set, but it's the first one I "noticed" as I have also been using them a lot on other projects 😁 )

Reply:
I have removed f-string usage for now, since v1.4.0 still support python 3.5

Reply:
Now that we are on release 2.0 we can make a release that officially breaks Python 3.6 and unlock f-strings. I'd rather close this PR before then, so thanks for changing.